Broken in 296.1

Fix it Mister Rhadamus or I will devour your offspring and then pull request a fix myself
```
> Task :app:compileDebugJavaWithJavac
C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:93: error: method allocNative2 in class CImage cannot be applied to given types;
            allocNative2((MMFRuntime.inst.app.hdr2Options & 4096) != 0, (short)-1, CServices.getBitmapPixels(createBitmap), 0, 0, 0, 0, createBitmap.getWidth(), createBitmap.getHeight(), SurfaceView.ES);
            ^
  required: int,int[],int,int,int,int,int,int,int
  found:    boolean,short,int[],int,int,int,int,int,int,int
  reason: actual and formal argument lists differ in length
C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:180: error: method readFrameToTexture in class GLRenderer cannot be applied to given types;
        GLRenderer.inst.readFrameToTexture(imageTexture, objX, objY, objWidth, objHeight);
                       ^
  required: ITexture,int,int,int,int,int,boolean
  found:    CImage,int,int,int,int
  reason: actual and formal argument lists differ in length
C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:232: error: method renderImage in class GLRenderer cannot be applied to given types;
            GLRenderer.inst.renderImage(imageTexture, resample, objX, objY, objWidth, objHeight, 0, 0);
                           ^
  required: ITexture,boolean,int,int,int,int,int,int,int
  found:    CImage,boolean,int,int,int,int,int,int
  reason: actual and formal argument lists differ in length
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
3 errors

> Task :app:compileDebugJavaWithJavac FAILED

[Incubating] Problems report is available at: file:///C:/Users/Alone/AppData/Local/Temp/And86B0.tmp/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler output below.
  C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:93: error: method allocNative2 in class CImage cannot be applied to given types;
              allocNative2((MMFRuntime.inst.app.hdr2Options & 4096) != 0, (short)-1, CServices.getBitmapPixels(createBitmap), 0, 0, 0, 0, createBitmap.getWidth(), createBitmap.getHeight(), SurfaceView.ES);
              ^
    required: int,int[],int,int,int,int,int,int,int
    found:    boolean,short,int[],int,int,int,int,int,int,int
    reason: actual and formal argument lists differ in length
  C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:180: error: method readFrameToTexture in class GLRenderer cannot be applied to given types;
          GLRenderer.inst.readFrameToTexture(imageTexture, objX, objY, objWidth, objHeight);
                         ^
    required: ITexture,int,int,int,int,int,boolean
    found:    CImage,int,int,int,int
    reason: actual and formal argument lists differ in length
  C:\Users\Alone\AppData\Local\Temp\And86B0.tmp\app\src\main\java\Extensions\CRunPerspective.java:232: error: method renderImage in class GLRenderer cannot be applied to given types;
              GLRenderer.inst.renderImage(imageTexture, resample, objX, objY, objWidth, objHeight, 0, 0);
                             ^
    required: ITexture,boolean,int,int,int,int,int,int,int
    found:    CImage,boolean,int,int,int,int,int,int
    reason: actual and formal argument lists differ in length
  Note: Recompile with -Xlint:deprecation for details.
  Note: Recompile with -Xlint:unchecked for details.
  Note: Some input files use or override a deprecated API.
  Note: Some input files use unchecked or unsafe operations.
  3 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to get full insights.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.14.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 11s
32 actionable tasks: 32 executed
```

--- a/Android/app/src/main/java/Extensions/CRunPerspective.java
+++ b/Android/app/src/main/java/Extensions/CRunPerspective.java
@@ -90,7 +90,8 @@ public class CRunPerspective extends CRunExtension {
     public class CImageTexture extends CImage {
         public CImageTexture(int i, int i2) {
             Bitmap createBitmap = Bitmap.createBitmap(i, i2, Bitmap.Config.ARGB_8888);
-            allocNative2((MMFRuntime.inst.app.hdr2Options & 4096) != 0, (short)-1, CServices.getBitmapPixels(createBitmap), 0, 0, 0, 0, createBitmap.getWidth(), createBitmap.getHeight(), SurfaceView.ES);
+            allocNative2(-1, CServices.getBitmapPixels(createBitmap), 0, 0, 0, 0, createBitmap.getWidth(), createBitmap.getHeight(), SurfaceView.ES);
+            setResampling((MMFRuntime.inst.app.hdr2Options & 4096) != 0);
         }
 
         @Override // Banks.CImage
@@ -177,7 +178,7 @@ public class CRunPerspective extends CRunExtension {
         int objWidth = ho.hoImgWidth;
         int objHeight = ho.hoImgHeight;
 
-        GLRenderer.inst.readFrameToTexture(imageTexture, objX, objY, objWidth, objHeight);
+        GLRenderer.inst.readFrameToTexture(imageTexture, objX, objY, objWidth, objHeight, 0xFFFFFF, true);
 
         if (imageTexture == null) {
             return;
@@ -229,7 +230,7 @@ public class CRunPerspective extends CRunExtension {
             }
 
             //imageTexture.setResampling(resample);
-            GLRenderer.inst.renderImage(imageTexture, resample, objX, objY, objWidth, objHeight, 0, 0);
+            GLRenderer.inst.renderImage(imageTexture, resample, objX, objY, objWidth, objHeight, 0, 0, 0);
             GLRenderer.inst.removeEffectShader();
             GLRenderer.inst.popClip();
         } }

